CONTRIBUTING.md

```diff
@@ -136,14 +136,14 @@ However, there are still some styles that the linter cannot pick up. If you are
 
 ## Git Branch Organization
 
-Submit all changes directly to the [`master branch`](https://github.com/facebook/react/tree/main). We don’t use separate branches for development or for upcoming releases. This requires us to always keep the `master` branch in a deployable state which means:
+Submit all changes directly to the [`master branch`](https://github.com/streetwriters/notesnook). We don’t use separate branches for development or for upcoming releases. This requires us to always keep the `master` branch in a deployable state which means:
 
 1. All tests must be passing at all times
 2. There should be as few breaking changes as possible
 
-## **Review Process**
+## Review Process
 
-We deeply appreciate everyone who takes the time to make a contribution. We will review all contributions as quickly as possible. **As a reminder, **[**opening an issue**](https://github.com/opensearch-project/OpenSearch/issues/new/choose)** discussing your change before you make it is the best way to smooth the PR process.** This will prevent a rejection because someone else is already working on the problem, or because the solution is incompatible with the architectural direction.
+We deeply appreciate everyone who takes the time to make a contribution. We will review all contributions as quickly as possible. **As a reminder, [opening an issue](https://github.com/opensearch-project/OpenSearch/issues/new/choose) and discussing your change before you make it, is the best way to smooth the PR process.** This will prevent a rejection because someone else is already working on the problem, or because the solution is incompatible with the architectural direction.
```

Comment:
Hey, the link should point to Notesnook here.

Reply:
Yeah, updated the url.